### PR TITLE
docs: vm upgrade using terraform

### LINKED ISSUE #5381: doc: add a method of upgrading for VM users
A rough outline of the process :
- Keep old VM installation running
- Install on a new VM the latest
- terraform import global configurations from the new installation
- make all custom IDP configurations via terraform .
- apply custom terraform configurations to the new installation. 

At this point there should be two versions up for example 1.0.13 and 1.0.14

- Traffic should be switched gradually from the old setup to the new setup.
- Once confidence is gained drain old VM.

A note needs to be added to the VM docs

```

We recommend using cloud-native deployments over VM deployments. VM deployments doesn't provide automatic upgrade/update support. Uninstalling and re-installing with newer binaries is the only option which requires re-configuration of auth server. Though this is made easy using t[erraform](Place link to the doc here). We recommend using cloud-native installations to avail smooth upgrades and better HA support

```

### PR DESCRIPTION
**Pull request by misba7**
### Prepare

- [ ] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [ ] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #5381 

#### Implementation Details
  

-------------------
### Test and Document the changes
- [ ] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

### CODE REVIEW

**Comment by mo-auto** [comment]
Error: Hi @misba7, You did not reference an open issue in your [PR](5551). I attempted to create an [issue](#5552) for you.    
 Please update that [issues'](#5552) title and body and make sure I correctly referenced it in the above PRs body.

File: docs/admin/vm-ops/upgrade.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+This guide shows how to upgrade a Janssen VM deployment.
+
+!!! Note
+    VM deployments don't provide automatic upgrade/update support. 
+    Uninstalling and re-installing with newer binaries is the only option that requires re-configuration of auth-server. 
+    Though this is made easy using [Terraform](../terraform/README.md). 
+    We recommend using Kubernetes installations over VM, to avail smooth upgrades and better HA support.
+
+
+Let's assume we are upgrading Jans VM installation from version `1.0.14` to `1.0.15`
```
Comment (moabu) [inline docs/admin/vm-ops/upgrade.md]: rephrase the assumption to remove `1.0.14` . You can simply rephrase it to `current version` to `vreplace-janssen-version`

File: docs/admin/vm-ops/upgrade.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Keep the old VM installation running.
+
+2. [Install](../install/vm-install/README.md) on a separate VM the target new Jans installation, i.e. 1.0.15.
```
Comment (moabu) [inline docs/admin/vm-ops/upgrade.md]: replace all `1.0.15` with `vreplace-janssen-version`

**Review by moabu — CHANGES_REQUESTED** [review]

**Review by moabu — APPROVED** [review]
